### ISSUE TITLE
Indent check is too strict in multiline function calls with array parameters

### ISSUE THREAD

**Issue by rkrisztian** [eslint/eslint#7373]
**Tell us about your environment**

* **ESLint Version:** 3.7.1
* **Node Version:** 6.2.2
* **npm Version:** 2.15.9

**What parser (default, Babel-ESLint, etc.) are you using?**

Default (did not specify).

**Please show your full configuration:**

```
{
  "env": {
    "amd": true,
    "jasmine": true
  },
  "rules": {
    "no-plusplus": "error",
    "no-caller": "error",
    "wrap-iife": "error",
    "no-extend-native": "error",
    "no-mixed-spaces-and-tabs": [
      "error",
      "smart-tabs"
    ],
    "camelcase": "error",
    "comma-style": "error",
    "dot-notation": "error",
    "one-var-declaration-per-line": "error",
    "new-cap": [
      "error",
      {
        "properties": false
      }
    ],
    "indent": [
      "error",
      "tab",
      {
        "SwitchCase": 1
      }
    ]
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

I use double indentation for continuation, so that I do not confuse one multiline statement as multiple ones:

```js
someObj.doSomething([
		'path/to/dependency1',
		'path/to/dependency2'
	],
	function (dependency1, dependency2) {
		dependency2.doSomethingElse(1);
		done();
	});
```

**What did you expect to happen?**

No errors reported by ESLint, i.e. source above is valid.

**What actually happened? Please include the actual, raw output from ESLint.**

```
/tmp/test/test.js
  2:3  error  Expected indentation of 1 tab but found 2   indent
  3:3  error  Expected indentation of 1 tab but found 2   indent
  4:2  error  Expected indentation of 0 tabs but found 1  indent
```

With --fix, file got reformatted like this:

```js
someObj.doSomething([
	'path/to/dependency1',
	'path/to/dependency2'
],
	function (dependency1, dependency2) {
		dependency2.doSomethingElse(1);
		done();
	});
```

Formerly I used JSCS 1.31.1, where this kind of code formatting was accepted.

**Comment by not-an-aardvark** [comment]
Thanks for the report.

However, I think this is a duplicate of https://github.com/eslint/eslint/issues/6990. Could you follow that issue for updates instead?

**Comment by not-an-aardvark** [comment]
Closing this issue as a duplicate, but please feel free to continue the discussion or reopen it if you disagree.
